CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
+# Note: Call the cmake_minimum_required() command at the beginning of the top-level CMakeLists.txt
+# file even before calling the project() command. The cmake_minimum_required(VERSION) command
+# implicitly invokes the cmake_policy(VERSION) command to specify that the current project
+# code is written for the given range of CMake versions.
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
The `(CMAKE_)CXX_STANDARD(_REQUIRED)` are in cmake from v3.1 :-(

Reply:
bah - that will be the funny part - i was a bit surprised and pissed when i see that the panel was compiled requiring 3.6.2 but then fail to link to some plugins correctly - like the volume or keyboard plugin. 

We should debug it - seems that some relevant policies are changed.